private/buf/bufwkt/cmd/wkt-go-data/main.go

```diff
@@ -282,17 +289,26 @@ const Version = "`)
 		p(path)
 		p(`": {
 `)
-		data := pathToData[path]
+		rawData := pathToData[path]
+		var buf bytes.Buffer
+		compressor := gzip.NewWriter(&buf)
```

Comment:
Compression is a little controversial - would be interested if there's any hit on perf with WKTs (likely) and what the binary size diff is

Reply:
Binary is 64k smaller and any difference in performance is negligible/de minimis (decompressing things this small is super fast).

**before**:
```
$ ls -l buf
-rwxr-xr-x  1 jhumphries  staff  58248834 Apr 25 22:21 buf
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 78% cpu 0.025 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 81% cpu 0.024 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 74% cpu 0.029 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 80% cpu 0.025 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.02s user 0.01s system 78% cpu 0.036 total
```

**after**:
```
$ ls -l buf
-rwxr-xr-x   1 jhumphries  staff  58182946 Apr 25 22:19 buf
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 95% cpu 0.020 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.02s user 0.01s system 85% cpu 0.030 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.02s user 0.01s system 90% cpu 0.029 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.01s user 0.01s system 89% cpu 0.020 total
$ time ./buf --version
1.32.0-beta.2-dev
./buf --version  0.02s user 0.01s system 80% cpu 0.033 total
```

Reply:
Removed the gzipping.